src/stores/EventsStore.tsx

```diff
@@ -24,11 +24,12 @@ export class EventsStore {
 
     setEvent<K extends keyof IEvent, V extends IEvent[K]>(key: K, value: V) {
         this.event[key] = value
-        this.eventSchema.validate(this.event)
+        this.eventSchema.validateKey(key, this.event)
     }
 
     async createEvent() {
         this.eventSchema.validate(this.event)
+        this.isButtonDisabled = true
```

Comment:
probaj poimenovat spremenljivke bolj smiselno. Imena naj ne bodo povezana med store in komponentami. 

Store neve da boš v komponenti ti button disablal. Tak da tukaj je smiselno poimenovat spremenljivko recimo `isSaving`.